tests/message/test_concents.py

```diff
@@ -0,0 +1,171 @@
+import unittest
+
+from golem_messages import message
+from golem_messages.message import concents
+
+from tests import factories
+
+class ConcentsTest(unittest.TestCase):
+
+    def test_subtask_result_verify(self):
+        srr = factories.SubtaskResultsRejectedFactory()
+        msg = factories.SubtaskResultsVerifyFactory(
+            slots__subtask_result_rejected=srr,
+        )
+        expected = [
+            ['subtask_result_rejected', srr]
+        ]
+
+        self.assertEqual(expected, msg.slots())
+        self.assertIsInstance(msg.subtask_result_rejected,
+                              message.tasks.SubtaskResultsRejected)
+
+    def test_ack_subtask_result_verify(self):
+        srv = factories.SubtaskResultsVerifyFactory()
+        msg = factories.AckSubtaskResultsVerifyFactory(
+            slots__subtask_results_verify=srv,
+        )
+        expected = [
+            ['subtask_result_verify', srv]
+        ]
+
+        self.assertEqual(expected, msg.slots())
+        self.assertIsInstance(msg.subtask_result_verify,
+                              concents.SubtaskResultsVerify)
+
+    def test_subtask_result_settled_no_acceptance(self):
+        ttc = factories.TaskToComputeFactory()
+        msg = factories.SubtaskResultsSettledFactory.origin_acceptance_timeout(
+            slots__task_to_compute=ttc
+        )
+        expected = [
+            ['origin',
+             concents.SubtaskResultsSettled.Origin.ResultsAcceptedTimeout
+             .value],
+            ['task_to_compute', ttc]
+        ]
+
+        self.assertEqual(expected, msg.slots())
```

Comment:
You could use `self.assertCountEqual()` and don't care about the ordering of slots.

Reply:
@jiivan hmm, doesn't the equality check verify also the contents?

Reply:
@shadeofblue https://docs.python.org/3/library/unittest.html#unittest.TestCase.assertCountEqual